### ISSUE TITLE
Is there a way to set the max number of rows?

### ISSUE THREAD

**Issue by irfaan** [tsdmrfth/react-native-steve#24]
Currently, the example shows 3 rows based on the number of items in `topics`. Is the number of rows configurable?

**Comment by tsdmrfth** [comment]
No. The component is designed to render all given children

**Comment by irfaan** [comment]
No, what I mean is - render all given children, but define how many rows it will span (e.g. force it to max 2 rows).

**Comment by tsdmrfth** [comment]
Ah, okay. I got you now. I need to think about this but not sure we can achieve this (for now).

**Comment by codedpills** [comment]
> Ah, okay. I got you now. I need to think about this but not sure we can achieve this (for now).

Any update on this?
